docs/general/commands.md

```diff
@@ -66,11 +66,11 @@ npm run start:production
 Starts the production server, building the app for optimal performance: assets are
 minified and served gzipped.
 
-### Port
+### Host and Port
 
-To change the port the app is accessible at pass the `--port` option to the command
-with `--`. E.g. to make the app visible at `localhost:5000`, run the following:
-`npm start -- --port 5000`
+To change the host and/or port the app is accessible at pass the `--host` and `--port` option to the command
```

Comment:
Maybe a comma after "accessible at" and put another "and/or" between --host --port.

`To change the host and/or port the app is accessible at, pass the `--host` and/or `--port` option(s) to the command`